build.psm1

```diff
@@ -468,6 +468,16 @@ cmd.exe /C cd /d "$location" "&" "$($vcVarsPath)\vcvarsall.bat" "$NativeHostArch
         }
         New-Item -Path $refDestFolder -ItemType Directory -Force -ErrorAction Stop > $null
         Copy-Item -Path $refAssemblies -Destination $refDestFolder -Force -ErrorAction Stop
+
+        # create the symlinks needed on Linux
+        if ($IsRedHatFamily) {
+            if ( ! (test-path "$publishPath/libssl.so.1.0.0")) {
```

Comment:
a symlink can be created if the target doesn't exist, we can check but it's not really needed